src/Magnum/BulletIntegration/Test/IntegrationTest.cpp

```diff
@@ -29,14 +29,29 @@
 #include <Corrade/TestSuite/Tester.h>
 
 #include <Magnum/Magnum.h>
+
+#include <Magnum/Math/Vector3.h>
+
+#include <Magnum/SceneGraph/MatrixTransformation3D.h>
+#include <Magnum/SceneGraph/Object.h>
+#include <Magnum/SceneGraph/Scene.h>
+
 #include "Magnum/BulletIntegration/Integration.h"
 #include "Magnum/BulletIntegration/DebugDraw.h"
+#include "Magnum/BulletIntegration/MotionState.h"
+
+#include <BulletDynamics/btBulletDynamicsCommon.h>
```

Comment:
(Okay, I'm just being annoying here. Sorry.)

I would not separate the includes with empty lines. Just two groups -- one with `<>` and the second with `""`.

EDIT: I saw you fixed that in a later commit -- could you squash those two together?